packages/wrangler/src/publish/list.ts

```diff
@@ -0,0 +1,35 @@
+import { fetchResult } from "../cfetch";
+import { logger } from "../logger";
+
+export type ServicesListRes = [
+	{
+		id: string;
+		created_on: string;
+		default_environment: {
+			script: {
+				last_deployed_from: string;
+			};
+		};
+	}
+];
```

Comment:
```suggestion
export type ServicesListRes = {
		id: string;
		created_on: string;
		default_environment: {
			script: {
				last_deployed_from: string;
			};
		};
	};
```
And then use as `ServicesListRes[]`